.git-blame-ignore-revs

```diff
@@ -0,0 +1,3 @@
+894744537dc64f729a44f64fbcb58e6b3b5dc1aa
```

Comment:
Since we're squashing commits, there should only be one entry per PR. Can you check if it's the case that the hash of the first commit is used after the squash?

Reply:
I was actually wondering about this and meant to check how it works, and it seems the above does [not](https://github.com/orgs/community/discussions/16880). It seems that we'll have to do two separate PRs one with the changes and one with the relevant commits in `.git-blame-ignore-revs`.

Reply:
Well I could squash the PR commits to one and then add another commit before merging into master instead of squashing?